.github/workflows/terraform-on-branch-push.yml

```diff
@@ -0,0 +1,79 @@
+name: Terraform tools on Branch Push
+on:
+  push:
+    branches-ignore: 
+      - main
+jobs:
+   docs:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+    steps:
+    - uses: actions/checkout@v3
```

Comment:
We should use the latest version, `actions/checkout@v4` here.

Reply:
fixed